docs/developer/payments/price-calculations.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Price calculations
+---
+
+This document provides an overview of how various transaction amount fields in a `TransactionItem` are calculated and influenced. Additionally, it provides details on how the totalBalance is calculated for Checkout and Order entities.
+
+## Transaction Amount Fields Overview
+
+The following fields are calculated based on transaction events and their types:
+
+- **`authorizedAmount`**: The total amount successfully authorized.
+- **`chargedAmount`**: The total amount successfully charged.
+- **`refundedAmount`**: The total amount successfully refunded.
+- **`canceledAmount`**: The total amount successfully canceled.
+- **`authorizePendingAmount`**: The total amount of pending authorizations.
+- **`chargePendingAmount`**: The total amount of pending charges.
+- **`refundPendingAmount`**: The total amount of pending refunds.
+- **`cancelPendingAmount`**: The total amount of pending cancellations.
+
+### Assumptions
```

Comment:
suggestion: I think it would be easier to read if we put Assumptions section **after** a brief explanation how does the amount calculation even works, what this is.

```suggestion
### Summary

When the transaction amounts are recalculated, firstly all values are set to zero, 
then the calculations are applied in the following order:
  1. recalculate amounts based on events without PSP reference
  2. recalculate amounts based on `AUTHORIZATION` events
  3. recalculate amounts based on `CHARGE` events
  4. recalculate amounts based on `REFUND` events
  5. recalculate amounts based on `CANCEL` events

### Assumptions
```